### ISSUE TITLE
UI improvements

### ISSUE THREAD

**Issue by giosce** [codeforamerica/brigade-project-index-statusboard#161]
I think it would be good to:
Move the sorting button outside of the filter box
Make last-issues and last-pushed columns narrower
Align the topics/tags on the left
How can sort by open-issues and last-pushed be shown on phone?

**Comment by rkaufman13** [comment]
Is this a duplicate of #154 ?

**Comment by giosce** [comment]
> Is this a duplicate of #154 ?

No, if you see the description this is about other smaller UI improvements.
